### PR TITLE
Add GA/UA Tracking on the backend

### PR DESCRIPTION
**Pull request by markolson**
Pushing these events up to GA should let us quickly correlate the number of fronted submissions each day with the number of backend transmissions each night.

### CODE REVIEW

**Comment by aub** [comment]
We're currently using Prometheus to track some similar metrics in Rx and SM. Do we have thoughts on when we should use that vs GA? Is there a specific reason to use GA in this case?

**Comment by markolson** [comment]
@aub I'd like to use GA because that's where fronted events on submissions (success and failure) are being tracked, and tracking backend events along side the frontend events would, I think, make it easy for business owners to eyeball that things were working as expected.

**Comment by aub** [comment]
Cool, yeah I'm not at all against GA. Just don't want to have a different tracking mechanism for every little thing. This LGTM.

**Comment by markolson** [comment]
Me either. I'll probably add a `StatsD.gauge("education_benefits.spool_file_written.rpo_#{region_id}", records.count)` or something in a PR later this week after talking about namespacing these sort of metrics in Prometheus.

File: config/google_analytics.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+global: 'UA-50123418'
+education_benefits: 'UA-50123418-7'
```
Comment (markolson) [inline config/google_analytics.yml]: These are both 'prod' keys - one that's for the global vets.gov account, another that's just for the education benefits form. I initially had these as environment variables in application.yml, but started thinking that a distinct config file made more sense given the possible number of tokens involved in the future. There is also a test/sandbox GA environment that could be the default one, and then the tokens in this PR are written down in the overlay..

**Comment by leannammiller** [comment]
@markolson what do you want to do on this one?

**Comment by markolson** [comment]
@knkski if I wanted to have different `config/google_analytics.yml` files between prod and staging/dev, how would I go about that?

**Pull request closed without merging**